### DISCUSSION TITLE
Error 400: redirect_uri_mismatch

### DISCUSSION THREAD

**Discussion by DrMxrcy** [outline/outline#4035] [Self Hosting] [answered]
Hi. I am trying to setup outline. When I try to login into google, it give me a Error 400: redirect_uri_mismatch.

The error is 

redirect_uri: https://outline.domain.com/outline.domain.com/auth/google.callback.

**Comment by cmpnnt** [comment, marked as answer]
I was just struggling with this error, and I finally solved it. For me, the error was caused by failing to put the protocol (https://) in front of the FQDN in the `url` environment variable.

One thing I'm noticing about yours is that you have `domain.outline.com` in your uri twice. Is that intentional? Shouldn't it be: `https://outline.domain.com/auth/google.callback`?
